src/presentation/test/integration/registration/limitedPartner/add-limited-partner-person.test.ts

```diff
@@ -16,6 +16,8 @@ import { TERRITORY_CHOICE_LIMITED_PARTNER_USUAL_RESIDENTIAL_ADDRESS_URL } from "
 import {
   PartnershipType
 } from "@companieshouse/api-sdk-node/dist/services/limited-partnerships";
+import { REGISTRATION_BASE_URL } from "config";
+import { LIMITED_PARTNER_CHOICE_TEMPLATE, REVIEW_LIMITED_PARTNERS_TEMPLATE } from "presentation/controller/registration/template";
```

Comment:
the import should have the ../../ in the path ( i know it works without in the tests but best to keep it same as the others)